frontend/public/features.ts

```diff
@@ -117,11 +119,21 @@ const detectCanCreateProject = dispatch => coFetchJSON(projectRequestPath)
     }
   );
 
+const serviceCatalogPath = `${k8sBasePath}/apis/servicecatalog.k8s.io/v1beta1/clusterserviceclasses?limit=1`;
+const detectServiceCatalogAvailable = dispatch => coFetchJSON(serviceCatalogPath)
```

Comment:
We already know if the types exist, so we don't need a new request. We should follow the pattern used for CRDs.

https://github.com/openshift/console/blob/master/frontend/public/features.ts#L55-L60